internal/configs/.stylelintrc.json

```diff
@@ -1,35 +1,28 @@
 {
-  "extends": ["stylelint-config-sass-guidelines"],
+  "extends": ["stylelint-config-sass-guidelines", "stylelint-config-prettier"],
   "plugins": ["stylelint-use-logical"],
   "rules": {
-    "color-hex-case": "upper",
```

Comment:
Just curious, how did you determine/decide which rules are not needed anymore?

Reply:
i removed formatting-related rules (which are now [deprecated](https://stylelint.io/migration-guide/to-15#deprecated-stylistic-rules)), because prettier will take care of formatting.